src/shared/components/FileUpload/FileUpload.tsx

```diff
@@ -153,14 +163,11 @@ const FileUpload: FunctionComponent<FileUploadProps> = ({
 					</Spacer>
 				);
 			}
+			const queryParams = queryString.parse(url.split('?').pop() || '');
+			const title: string = get(queryParams, 'name', 'bestand') as string;
 			return (
 				<Spacer margin="bottom-small" key={url}>
-					<Blankslate
-						title={url.split('/').pop() || ''}
-						body=""
-						icon="file"
-						className="a-upload-file-preview"
-					>
+					<Blankslate title={title} body="" icon="file" className="a-upload-file-preview">
```

Comment:
Kunnen we `body` dan niet gewoon optioneel maken op component-niveau?

Reply:
https://github.com/viaacode/avo2-components/pull/241